app/role_data.go

```diff
@@ -25,33 +25,49 @@ var defaultAdminLabel = map[string]string{defaultAdminLabelKey: defaultAdminLabe
 func addRoles(management *config.ManagementContext) (string, error) {
 	rb := newRoleBuilder()
 
-	rb.addRole("Create Clusters", "clusters-create").addRule().apiGroups("management.cattle.io").resources("clusters").verbs("create").
+	rb.addRole("Create Clusters", "clusters-create").
+		addRule().apiGroups("management.cattle.io").resources("clusters").verbs("create").
```

Comment:
Is this adding an extra line?

Reply:
not an extra line, just moving to next line. (the extra space is due to the browser, if you zoom out, you can see there is not extra line)

Reply:
yep, looks ok in zoomed out view